src/core/tsi/ssl_transport_security_utils.h

```diff
@@ -142,6 +142,23 @@ tsi_result SslProtectorUnprotect(const unsigned char* protected_frames_bytes,
                                  unsigned char* unprotected_bytes,
                                  size_t* unprotected_bytes_size);
 
+// Verifies the CRL signature using X509_CRL_verify
+// https://www.openssl.org/docs/man3.0/man3/X509_CRL_verify.html
+// return: 1 if valid, 0 if invalid, and -1 is there was a problem
+int verify_crl_signature(X509_CRL* crl, X509* issuer);
```

Comment:
For this function and the ones below, can we use C++-style naming, e.g. `VerifyCrlSignature`?

Reply:
Definitely, don't know what I was thinking with the naming 🤦